src/api-explorer/v4-0/ConcurRequest.swagger2.json

```diff
@@ -0,0 +1,2562 @@
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "requestUuid",
+            "in": "path",
+            "description": "The unique identifier of the Request",
+            "required": true,
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "201": {
+            "description": "Created",
+            "schema": {
+              "$ref": "#/definitions/ResourceLink"
+            }
+          },
+          "400": {
+            "description": "Bad param: invalid Guid",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          },
+          "401": {
+            "description": "Invalid or non existent authorization HTTP header",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          },
+          "403": {
+            "description": "Permission Denied",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          },
+          "404": {
+            "description": "Not found",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          },
+          "408": {
+            "description": "Time out",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          },
+          "500": {
+            "description": "Internal server error",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          },
+          "503": {
+            "description": "The service is currently unavailable",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/requests/{requestUuid}/{action}": {
+      "post": {
+        "tags": [
+          "Workflow Resource"
+        ],
+        "summary": "Move an existing Request in the approval workflow",
+        "operationId": "workflowActionUsingPOST",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "requestUuid",
+            "in": "path",
+            "description": "The unique identifier of the Request",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "action",
+            "in": "path",
+            "description": "The workflow state transition to be executed",
+            "required": true,
+            "type": "string",
+            "enum": [
+              "submit",
+              "approve",
+              "recall",
+              "sendback",
+              "cancel",
+              "close",
+              "reopen"
+            ]
+          },
+          {
+            "in": "query",
```

Comment:
This is a body field apparently. Check why it is generated as a query param.

Reply:
Charlotte valid the 'String' type of comment, to keep it the same as reference.

Reply:
Hi Quancheng, is it a question for me? if yes the comment is a string as all other comment fields of the other Resources (only the new comments fields developed recently are fields linked to a href towards the new Request comments & Expense comments endpoints